index.js

```diff
@@ -28,7 +28,6 @@ module.exports = {
       type: 'textarea',
       placeholder: 'type here...',
       settings: {
-        empty: true,
         minlength: '50',
```

Comment:
because then this requirement could be enforced by a browser which uses those, and may have javascript turned off, to limit the user's ability to write too much.